src/Pipe/index.tsx

```diff
@@ -60,3 +60,5 @@ const Pipe: React.ComponentType<{}> & { Item: typeof PipeItem } = ({ children })
 );
 
 Pipe.Item = PipeItem;
+
+export { Pipe };
```

Comment:
This breaks when testing on localhost
`TypeError: Cannot read properties of undefined (reading 'Item')`